vllm/worker/hpu_model_runner.py

```diff
@@ -1900,6 +1900,8 @@ def _prepare_prompt(
             # prefix caching
 
             max_num_block = max(len(bt) for bt in prefix_block_tables)
```

Comment:
max_num_block = max(len(bt) for bt in prefix_block_tables)
this line is not needed since you calc it again.